main.go

```diff
@@ -277,6 +282,8 @@ func commandLine() commandLineArgs {
 		"during 'prune' process this many transactions together")
 	flags.IntVar(&pruneSleepTimeMs, "prune-batch-sleep-ms", 0,
 		"during 'prune' sleep this long between batches (reduces load)")
+	// flags.BoolVar(&deleteInvalidTxns, "delete-invalid-txns", false,
```

Comment:
I assume this was for an experiment?

Reply:
This was along the same lines as the latter one. We have some patterns (eg metrics going missing), but I didn't necessarily want to just force remove everything without giving feedback.
I suppose as long as we log what we are cleaning up (should we include the full txn description?)
Then it can get things working, but we can have something that we can debug what might have happened.
Thoughts?